jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -265,6 +266,19 @@ public void setConnectors(Connector[] connectors)
             _connectors.addAll(Arrays.asList(connectors));
     }
 
+    /**
+     * Add a {@link Connection.Listener} as a bean to all connectors on the server, this
+     * will register the listener with all connections accepted by the connectors.
+     * @param listener the listener to be added.
+     */
+    public void addToAllConnectors(Connection.Listener listener)
+    {
+        for (Connector connector : getConnectors())
+        {
+            connector.addBean(listener);
+        }
+    }
```

Comment:
```suggestion
    /**
     * Add a bean to all connectors on the server.
     * If the bean is an instance of {@link Connection.Listener} it will also be 
     * registered as a listener on all connections accepted by the connectors.
     * @param bean the bean to be added.
     */
    public void addBeanToAllConnectors(Object bean)
    {
        for (Connector connector : getConnectors())
        {
            connector.addBean(bean);
        }
    }
```